### PR TITLE
Add bubble wrap game to Games cog.

### PR DESCRIPTION
**Pull request by h-anjru**
`bubblewrap` is a simple fidget/stress relief game.

### CODE REVIEW

File: crimsobot/cogs/games.py [hunk 1]
```diff
@@ -598,6 +598,75 @@ async def daily(self, ctx: commands.Context, lucky_number: int) -> None:
...
+                await ctx.send(embed=error_embed, delete_after=18)
+                return
+
+        bubble = bubble.strip()
+
+        chars_to_remove = ['\n', '|', '`', '\u200b', '\u200d']
+
+        for char in chars_to_remove:
+            if char in bubble:
+                bubble = bubble.replace(char, '')
```
Comment (h-anjru) [inline crimsobot/cogs/games.py:658]: Justification for this method over previous method or regex--I chose fastest method described in the accepted answer from here: https://stackoverflow.com/questions/3411771/best-way-to-replace-multiple-characters-in-a-string

Comment (williammck) [reply crimsobot/cogs/games.py:658]: stack overflow ftw

File: crimsobot/cogs/games.py [hunk 2]
```diff
@@ -598,6 +598,75 @@ async def daily(self, ctx: commands.Context, lucky_number: int) -> None:
...
+        )
+
+        # check and clean the input
+        if bubble is None:
+            bubble = choose_random_emoji()
+        elif len(bubble) > char_limit:
+            # check if input is a custom emoji from the server
+            emoji_strings = []
+            for emoji in ctx.guild.emojis:
+                emoji_strings.append(str(emoji))
```
Comment (williammck) [inline crimsobot/cogs/games.py:647]: slight nit, but _list comprehensions are beautiful_:
(please imagine the italicized text is glittery)

```suggestion
            emoji_strings = [str(e) for e in ctx.guild.emojis]
```

File: crimsobot/cogs/games.py [hunk 3]
```diff
@@ -598,6 +598,75 @@ async def daily(self, ctx: commands.Context, lucky_number: int) -> None:
...
+        There is a per-channel cooldown on this command."""
+
+        def choose_random_emoji() -> str:
+            """Choose a random emoji from the emojis used in Emojistory."""
+            # read in lines of emojis
+            emojis = open(c.clib_path_join('games', 'emojilist.txt'), encoding='utf8', errors='ignore').read()
+            # strip newlines
+            emojis = emojis.replace('\n', '')
+            # choose and return
+            return random.choice(emojis)
```
Comment (williammck) [inline crimsobot/cogs/games.py:618]: i feel like this function being _here_ is weird but the current functionality isn't duplicated anywhere as of right now, so this is probably fine.
future refactor: `Emoji` class for getting random emoji and such?

File: crimsobot/cogs/games.py [hunk 4]
```diff
@@ -598,6 +598,75 @@ async def daily(self, ctx: commands.Context, lucky_number: int) -> None:
...
+                '· that emoji is not in this server,',
+                '· after cleaning up your input, there are no characters left, or',
+                f'· your input is too long. (Under {char_limit} characters pls!)',
+                '',
+                'Some Discord objects such as mentions have hidden characters that will make the input too long.'
+            ]),
+            thumb_name='weary',
+            color_name='yellow',
+            footer='Please try again!'
+        )
```
Comment (williammck) [inline crimsobot/cogs/games.py:638]: i like the thought behind this, but the engineer in me says that specific errors are always better...
maybe put it in a nested function and just pass in the error message where applicable?

File: crimsobot/cogs/games.py (hunk 3 above)
Comment (h-anjru) [reply crimsobot/cogs/games.py:618]: We can only re-use this functionality on one other place right now: `emojistory`. I guess we have to think about whether it's worth refactoring in the future as part of an `emojistory` refresh/revisit.

File: crimsobot/cogs/games.py (hunk 4 above)
Comment (h-anjru) [reply crimsobot/cogs/games.py:638]: This should eventually get wrapped into custom crimsoBOT errors. We've talked about it before, I guess it's almost time to do that. If you're ok with it, we can leave this here for now and move it to some crimsoBOT error module in the future?

Comment (williammck) [reply crimsobot/cogs/games.py:638]: works for me

**Pull request merged**